mender.go

```diff
@@ -574,6 +607,28 @@ func TransitionError(s State, action string) State {
 	return NewErrorState(me)
 }
 
+func shouldReportUpdateStatus(state MenderState) bool {
+	switch state {
+	case MenderStateUpdateInstall, MenderStateReboot,
+		MenderStateUpdateVerify, MenderStateUpdateCommit,
+		MenderStateAfterReboot, MenderStateRollback,
+		MenderStateRollbackReboot, MenderStateAfterRollbackReboot,
+		MenderStateUpdateError:
+		return true
```

Comment:
Does this mirror the contents of the `stateStatus` map? Can you iterate over that one instead and return true if it matches? The reason I'm asking is that this is the kind of thing that could easily get out of sync.

Reply:
Right. Good suggestion. I will change this one.